packages/sdk-demo/src/context/configContext/configContext.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { createContext, useContext, memo } from 'react';
+
+import { getConfig, ConfigSchema } from '@/core/getConfig';
+
+type ConfigContextType = ConfigSchema | null;
+
+const ConfigContext = createContext<ConfigContextType>(null);
```

Comment:
Don't mind renaming the file to `ConfigContext/ConfigContext.tsx`? I would also ask to check the `themeContext` too.